recipes/rpkmforgenes/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: "rpkmforgenes"
+  version: "1.0.1"
+
+build:
+  number: 0
+
+source:
+  git_url: https://github.com/danielramskold/S3_species-specific_sequencing
+  git_rev: 1.0.1
+  md5: foo
+
+requirements:
+  build:
+    - python >=2.7,<3
```

Comment:
This interferes with pinning (a python specific caveat). You can use `build->skip` instead.